src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRate = 40;
+  const baseCost = days * dailyRate;
+  let discount = 0;
```

Comment:
Prefer 'const' over 'let' for the discount variable. Since the discount value is not reassigned after its initial calculation, 'const' can be used to prevent unintentional changes, as per the checklist item 3.